packages/camltc/camltc.0.9.5/url

```diff
@@ -0,0 +1,2 @@
+git: "https://github.com/toolslive/camltc.git#64dd82ab1586b81b2939839bc2d3cba9ecee9a2a"
```

Comment:
in the opam-repository, we only want releases, not versions pointing to a git commit.. could you please do a release and point to a tarball and checksum.  thanks!

Reply:
The reason the git commit is used here is that the build uses a git submodule for tokyo cabinet.
So the build would need to be changed to fetch a tarbal iso a git submodule.
What's the reason you don't want versions pointing to a git commit ?
Btw, we've had this discussion before:
https://github.com/ocaml/opam-repository/pull/7661